src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1343 @@
+                    <option value="">Select</option>
+                    <option>Option 1</option>
+                    <option>Option 2</option>
+                    <option>Option 3</option>
+                  </select>
+                </div>
+                <div class="chi-form__item -mb--2">
+                  <label class="chi-label" for="example-ba14">Label</label>
+                  <select class="chi-select" id="example-ba14">
+                    <option value="">Select</option>
+                    <option>Option 1</option>
+                    <option>Option 2</option>
+                    <option>Option 3</option>
+                  </select>
+                </div>
+              </div>
+              <div class="-d--flex -justify-content--center -pb--2">
+                <button class="chi-button -primary -lg -uppercase -px--4 -no-hover">Apply</button>
+                <button class="chi-button -primary -outline -lg -bg--white -uppercase -px--4 -ml--2 -no-hover">Cancel</button>
+              </div>
             </div>
-            <div class="chi-drawer__content -px--2">
-              <ul class="-no-style">
-                <li class="-text--md">Download data</li>
-                <div class="chi-divider"></div>
-                <li class="-text--md">Refresh results</li>
-                <div class="chi-divider"></div>
-              </ul>
+          </div>
+        </div>
+        <div class="chi-backdrop -closed">
+          <div class="chi-backdrop__wrapper">
+            <div class="chi-drawer -right -menu -position--absolute" id="drawer-2">
+              <div class="chi-drawer__header">
+                <span class="chi-drawer__title">Actions</span>
+                <button class="chi-button -icon -close -no-hover" aria-label="Close">
+                  <div class="chi-button__content">
+                    <i class="chi-icon icon-x"></i>
+                  </div>
+                </button>
+              </div>
+              <div class="chi-drawer__content -px--2">
+                <ul class="-no-style">
+                  <li class="-text--md">Download data</li>
+                  <div class="chi-divider"></div>
+                  <li class="-text--md">Refresh results</li>
+                  <div class="chi-divider"></div>
+                </ul>
+              </div>
             </div>
           </div>
         </div>
-      </div>
 
-      <!-- Javascript -->
-      <script>
-        chi.drawer(document.getElementById('drawer-trigger-br1'));
-        chi.drawer(document.getElementById('drawer-trigger-br2'));
-        chi.tooltip(document.querySelectorAll('[data-tooltip]'));
-      </script>
+        <!-- Javascript -->
+        <script>
+          chi.drawer(document.getElementById('drawer-trigger-brp1'));
+          chi.drawer(document.getElementById('drawer-trigger-brp2'));
+          chi.tooltip(document.querySelectorAll('[data-tooltip]'));
+        </script>
+
+  // Drawers
```

Comment:
Drawers must appear in both mobile versions when cliking. Right now they are only visible in portal.

Reply:
True, solved.